assets/scss/base/form/_form.scss

```diff
@@ -0,0 +1,330 @@
+
+Markup:
+<fieldset class="{{modifier_class}}">
+  <div class="form-field">
+    <label for="first-name">
+      <input type="text" id="first-name" value=""/>
+      First Name
+    </label>
+  </div>
+ <div class="form-field">
+    <label for="second-name">
+      <input type="text" id="second-name" value=""/>
+      Second Name
+    </label>
+  </div>
+  <div class="form-field">
+    <button class="button" type="submit">Send</button>
+  </div>
+</fieldset>
+
+.form-field-group    - A form field group can be applied via `.form-field-group`
+.form-field-single   - A form field single can be applied via `.form-field-single`
+
+Styleguide Form.group
+*/
+
+
+fieldset {
+  @include core-19();
+  margin: 0;
+  position: relative;
+  width: 100%;
+  legend {
+    width: 100%;
+    margin-bottom: 0.75em;
+    &.h2-heading {
+      margin: 0 0 0.25em 0;
+    }
+  }
+}
+
+.form-field-group {
+  margin-bottom: 0.75em;
+  line-height: 1.6em;
+  @extend %contain-floats;
+  clear: both;
+}
+
+.form-field-single {
+  @extend .form-field-group;
+}
+
+
+/*
+Field
+
+A Form field is used to group several controls or elements within a web form.
+
+Markup:
+<fieldset class="form-field-group">
+  <div class="form-field {{modifier_class}}">
+    <label for="first-name">
+      <input type="text" id="first-name" value=""/>
+      First Name
+    </label>
+  </div>
+  <div class="form-field {{modifier_class}}">
+    <label for="second-name">
+      <input type="text" id="second-name" value=""/>
+      Second Name
+    </label>
+  </div>
+  <div class="form-field {{modifier_class}}">
+    <button class="button" type="submit">Send</button>
+  </div>
+</fieldset>
+
+.form-field--tall - A "tall" form field can be used via `.form-field--tall`
+
+Styleguide Form.field
+*/
+
+.form-field {
+  @extend %contain-floats;
+  float: left;
+  margin-bottom: 0.75em;
+  margin-top: 0;
+  width: 100%;
+}
+
+.form-field--tall {
+  margin: 0 0 em(30) 0;
+}
+
+
+/*
+Inline
+
+For an inline elements you have a few choices.
+
+Markup:
+<fieldset class="form-field-group">
+  <div class="{{modifier_class}}">
+    <label for="blue" class="block-label">
+      <input type="radio" id="blue"/>
+      Blue
+    </label>
+  </div>
+  <div class="{{modifier_class}}">
+    <label for="green" class="block-label">
+      <input type="radio" id="green"/>
+      Green
+    </label>
+  </div>
+  <div class="form-field">
+    <button class="button" type="submit">Send</button>
+  </div>
+</fieldset>
+
+.form-field--inline              - For an inline form field use `.form-field--inline`
+.form-field--inline-text-input   - For an inline form field for text input use `.form-field--inline-text-input`
+.form-field--inline-spaced-right - For an inline form field for text input spaced to the right use `.form-field--inline-spaced-right`
+.form-field--inline-spaced-left  - For an inline form field for text input spaced to the left use `.form-field--inline-spaced-left`
```

Comment:
The `BEM` here is confused. These are marked as modifiers of `form-field` but are actually children of `form-field` or seperate components - issue will be raised